cavedb/models.py

```diff
@@ -750,6 +750,8 @@ def __str__(self):
         return '%s (%s)' % (self.name, self.survey_id)
 
     class Meta:
+        verbose_name = 'Cave or Karst Feature'
+        verbose_name_plural = 'Caves and Karst Features'
```

Comment:
Nope!

When you're talking about a single object, it is either a Cave _or_ a Karst Feature.

When you're talking about all of them, they are collective Caves _and_ Karst Features.